charts/jindofs/templates/fuse/daemonset.yaml

```diff
@@ -13,6 +13,15 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
     role: jindofs-fuse
+  ownerReferences:
```

Comment:
The pods which created by DaemonSet/StatefulSet will not inherit the ownerReference of DaemonSet/StatefulSet. 

I wonder if this is as expected?

Reply:
Yes, it searches the controller OwnerReference Hierarchy from the pod. We can provide a common method to search the top owner.